### PR TITLE
changes to Blastoff

### PR DESCRIPTION
**Pull request by 0xheartcode**
Updating Token Info

### CODE REVIEW

**Comment by Define101** [comment]
https://github.com/DefiLlama/defillama-server/pull/7298
should be live within 1hour

File: defi/src/protocols/data3.ts [hunk 1]
```diff
@@ -34599,8 +34599,8 @@ const data3: Protocol[] = [
   {
     id: "4246",
     name: "Blastoff",
-    address: null,
-    symbol: "-",
+    address: "0xD55eDfc79c0d14084260D16f38BdA75e28AbFb6A",
```
Comment (g1nt0ki) [inline defi/src/protocols/data3.ts:34602]: can you change this to "blast:0x..."

Comment (0xheartcode) [reply defi/src/protocols/data3.ts:34602]: pushed

**Comment by Define101** [comment]
Its live @0xheartcode 
https://github.com/DefiLlama/defillama-server/pull/7298/files

**Pull request closed without merging**